suite-native/module-home/src/components/EmptyPortfolioCrossroads.tsx

```diff
@@ -0,0 +1,101 @@
+import { useNavigation } from '@react-navigation/native';
+
+import {
+    AccountsImportStackRoutes,
+    ConnectDeviceStackRoutes,
+    HomeStackParamList,
+    HomeStackRoutes,
+    RootStackParamList,
+    RootStackRoutes,
+    StackToStackCompositeNavigationProps,
+} from '@suite-native/navigation';
+import { VStack, Card, Button, Text, Pictogram } from '@suite-native/atoms';
+import { useNativeStyles, prepareNativeStyle } from '@trezor/styles';
+import { useTranslate } from '@suite-native/intl';
+import { useDeviceConnect } from '@suite-native/device';
+
+import { SettingsButtonLink } from './SettingsButtonLink';
+
+const cardStyle = prepareNativeStyle<{ flex: 1 | 2 }>((utils, { flex }) => ({
+    flex,
+    justifyContent: 'center',
+    paddingTop: 40,
+    paddingBottom: utils.spacings.extraLarge,
+}));
+
+type NavigationProps = StackToStackCompositeNavigationProps<
+    HomeStackParamList,
+    HomeStackRoutes.Home,
+    RootStackParamList
+>;
+
+export const EmptyPortfolioCrossroads = () => {
+    useDeviceConnect();
+    const { applyStyle } = useNativeStyles();
+    const navigation = useNavigation<NavigationProps>();
+
+    const { translate } = useTranslate();
+
+    const handleSyncMyCoins = () => {
+        navigation.navigate(RootStackRoutes.AccountsImport, {
+            screen: AccountsImportStackRoutes.SelectNetwork,
+        });
+    };
+
+    const handleConnectDevice = () => {
+        navigation.navigate(RootStackRoutes.ConnectDevice, {
+            screen: ConnectDeviceStackRoutes.ConnectAndUnlockDevice,
+        });
+    };
+
+    return (
+        <VStack spacing="medium" flex={1}>
+            <Card style={applyStyle(cardStyle, { flex: 2 })}>
+                <VStack spacing="large" justifyContent="center" alignItems="center">
+                    <Pictogram
+                        icon="trezor"
+                        variant="green"
+                        title={translate(
+                            'moduleConnectDevice.connectCrossroadsScreen.gotMyTrezor.title',
+                        )}
+                        subtitle={translate(
+                            'moduleConnectDevice.connectCrossroadsScreen.gotMyTrezor.description',
+                        )}
+                        size="large"
+                    />
+                    <Button onPress={handleConnectDevice} size="large">
+                        {translate(
+                            'moduleConnectDevice.connectCrossroadsScreen.gotMyTrezor.connectButton',
+                        )}
+                    </Button>
+                </VStack>
+            </Card>
+            <Card style={applyStyle(cardStyle, { flex: 1 })}>
+                <VStack spacing="large" justifyContent="center" alignItems="center">
+                    <VStack spacing="small" alignItems="center">
+                        <Text variant="titleSmall" textAlign="center">
+                            {translate(
```

Comment:
Prefer using Translation component if you don't need string specifically because you can easily add Rich Text Formatting later if needed

Reply:
https://github.com/trezor/trezor-suite/pull/9694/commits/c9f9b0587b4ba3bea0645ae010e508e0e0911fc4